examples/daymet-hi-annual.json

```diff
@@ -0,0 +1,186 @@
+        },
+        "y": {
+            "type": "spatial",
+            "axis": "y",
+            "description": "North American Lambert Conformal Conic, 1KM grid.",
+            "extent": [
+                -5802250,
+                -39000
+            ],
+            "step": 1000,
+            "reference_system": "PROJCRS[\"undefined\",BASEGEOGCRS[\"undefined\",DATUM[\"undefined\",ELLIPSOID[\"undefined\",6378137,298.257223563,LENGTHUNIT[\"metre\",1,ID[\"EPSG\",9001]]]],PRIMEM[\"Greenwich\",0,ANGLEUNIT[\"degree\",0.0174532925199433],ID[\"EPSG\",8901]]],CONVERSION[\"unknown\",METHOD[\"Lambert Conic Conformal (2SP)\",ID[\"EPSG\",9802]],PARAMETER[\"Latitude of 1st standard parallel\",25,ANGLEUNIT[\"degree\",0.0174532925199433],ID[\"EPSG\",8823]],PARAMETER[\"Latitude of 2nd standard parallel\",60,ANGLEUNIT[\"degree\",0.0174532925199433],ID[\"EPSG\",8824]],PARAMETER[\"Latitude of false origin\",42.5,ANGLEUNIT[\"degree\",0.0174532925199433],ID[\"EPSG\",8821]],PARAMETER[\"Longitude of false origin\",-100,ANGLEUNIT[\"degree\",0.0174532925199433],ID[\"EPSG\",8822]],PARAMETER[\"Easting at false origin\",0,LENGTHUNIT[\"metre\",1],ID[\"EPSG\",8826]],PARAMETER[\"Northing at false origin\",0,LENGTHUNIT[\"metre\",1],ID[\"EPSG\",8827]]],CS[Cartesian,2],AXIS[\"(E)\",east,ORDER[1],LENGTHUNIT[\"metre\",1,ID[\"EPSG\",9001]]],AXIS[\"(N)\",north,ORDER[2],LENGTHUNIT[\"metre\",1,ID[\"EPSG\",9001]]]]"
+        },
+        "nv": {
+            "type": "count?",
+            "description": "size of time_bnds",
+            "values": [
+                0,
+                1
+            ]
+        }
+    },
+    "cube:coordinates": {
+        "lat": {
+            "type": "latitude",
+            "extent": [
+                17.960035,
+                23.512327
+            ],
+            "description": "latitude coordinate",
+            "unit": "degrees_north",
+            "dimensions": [
+                "y",
+                "x"
+            ]
+        },
+        "lon": {
+            "type": "latitude",
+            "extent": [
+                -160.29884,
+                -154.77806
+            ],
+            "description": "longitude coordinate",
+            "unit": "degrees_east",
+            "dimensions": [
+                "y",
+                "x"
+            ]
+        }
+    },
+    "cube:variables": {
+        "prcp": {
+            "description": "The total accumulated precipitation over the monthly period of the daily total precipitation. Sum of all forms of precipitation converted to a water-equivalent depth.",
+            "extent": [
+                0,
+                null
+            ],
+            "unit": "mm",
```

Comment:
This probably isn't valid. Does STAC have a way to express it (maybe another extension we should use?)

Reply:
Why is this invalid? "mm" for millimeter sounds alright to me...

Reply:
Ah, sorry, I misunderstood this. Ignore me!